utils/deprecate_bot_prefixes.py

```diff
@@ -0,0 +1,21 @@
+from errbot import BotPlugin
+import config
+
+class DeprecateBotPrefixes(BotPlugin):
+    """
+    A callback for every message that starts with depecrated prefix, hence, leaving a deprecation notice
+    """
+
+    def callback_message(self, msg):
+        """
+        Notify the user issuing the command that use deprecated prefix.
+        """
+
+        for deprecated_prefix in config.BOT_DEPRECATED_PREFIXES:
+            if msg.body.startswith(deprecated_prefix+" "):
```

Comment:
I think `+" "` isnt needed here, as the prefix will already contain a ` `

Reply:
I got inspiration from how config define alternative for BOT_PREFIX. It use `'corobo '` instead of `'corobo'`. It respond to `'corobo command'` and not `'corobocommand'`. How about that?

Because i use whitespace for splitting env var, then i hardcoded space to separate prefix and command.

Reply:
Ah, i understand from above case. Thanks!